package.json

```diff
@@ -157,5 +157,13 @@
     "unist-util-visit": "^5.0.0",
     "uuid": "^9.0.1",
     "yaml": "^2.3.4"
+  },
+  "mocha": {
```

Comment:
Ah cool, then we can also remove these params in the `scripts['test:node']` for brevity right?